content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,144 @@
+.. image:: setup/amazon-accounts-link-setting.png
+   :align: center
+   :alt: The Amazon Accounts link beneath the Amazon Synch settings in Odoo Sales.
 
-#. Amazon redirects you to Odoo, with your account registered.
+Doing so reveals a separate :guilabel:`Amazon Accounts` page. From here, click :guilabel:`New` to
+create, and subsequently link, a new Amazon account.
 
-Once the account is registered, the marketplaces available to this account are synchronized and
-listed under the :guilabel:`Marketplaces` tab. If you wish, you can remove some items from the list
-of synchronized marketplaces to disable their synchronization.
+On the blank :guilabel:`Amazon Account` form page, start by choosing a name for the account (e.g.
+`American Marketplace`). Then, in the :guilabel:`Credentials` tab, select the desired marketplace on
+which the seller account was initially created from the :guilabel:`Home Marketplace` drop-down menu.
+
+.. image:: setup/amazon-accounts-form-page.png
+   :align: center
+   :alt: A typical Amazon Account form page in the Odoo Sales application.
+
+.. note::
+   After validation of credentials, the marketplaces to which this account has access is
+   synchronized, and automatically made available.
+
+After saving, the field in the :guilabel:`Credentials` tab is replaced by a :guilabel:`Link with
+Amazon` button.
+
+.. image:: setup/amazon-accounts-form-link-button.png
+   :align: center
+   :alt: A typical Amazon Account form page and Link with Amazon button in Odoo Sales.
+
+Clicking that button redirects to either the Amazon login page, or directly to the required consent
+page, if the user is already logged in to Amazon.
+
+On this page, confirm that Amazon is allowed to give Odoo access to the account and related data.
+
+Upon confirmation, Amazon returns the user to Odoo, and the account has been registered.
+
+With the account successfully registered, the marketplaces available to this specific account are
+synchronized and listed under the :guilabel:`Marketplaces` tab.
+
+If desired, remove items from the list of synchronized marketplaces to disable synchronization.
 
 Match database products in Amazon
 =================================
 
 When an Amazon order is synchronized, up to three sales order items are created in Odoo for each
-product sold on Amazon: one for the marketplace product, one for the shipping charges (if any) and
+product sold on Amazon: one for the marketplace product, one for the shipping charges (if any), and
 one for the gift wrapping charges (if any).
 
 .. _amazon/matching:
 
 The selection of a database product for a sales order item is done by matching its
-**internal reference** with the **SKU** for marketplace items, the **shipping code** for delivery
-charges, and the **gift wrapping** code for gift wrapping charges.
+:guilabel:`Internal Reference` with the *SKU* for marketplace items, the *Shipping Code* for
+delivery charges, and the *Gift Wrapping* code for gift wrapping charges.
```

Comment:
Great question. So, from what I gathered from the last doc on this, I do, indeed, believe the SKU, Shipping Code, and Gift Wrapping code are Amazon-related elements. I went ahead and clarified this a bit to avoid any confusion. I also added a quick explanation/reminder as to what an Internal Reference is in Odoo. I believe this should remedy the situation. However, if you think additional adjustments need to be made, just let me know. Thanks again for bringing this to my attention!